README.md

```diff
@@ -1,15 +1,28 @@
-# go2xunit
+# go2xunit [![Build Status](https://travis-ci.org/tischda/go2xunit.svg)](https://travis-ci.org/tischda/go2xunit)
 
 Converts `go test -v` (or `gocheck -vv`) output to xunit compatible XML output
 (used in [Jenkins][jenkins]/[Hudson][hudson]).
 
+This is a fork from go2xunit by Miki Tebeka <miki.tebeka@gmail.com>.
```

Comment:
Oh sorry, I have not read the documentation properly: "After your pull
request is sent, any new commits pushed to your branch will
automatically be added to the pull request.".

The pull request should have ended here:

commit 253d2d5b
Date:   Sat Jun 6 00:36:17 2015 +0200

    set assembly attribute

After that, I ripped everything to pieces to play around with your code
(I'm learning GO).
Do you want me to create another pull request ?


On 14/06/2015 07:34, Miki Tebeka wrote:
>
> In README.md
> <https://github.com/tebeka/go2xunit/pull/20#discussion_r32376588>:
>
> >  
> >  Converts `go test -v` (or `gocheck -vv`) output to xunit compatible XML output
> >  (used in [Jenkins][jenkins]/[Hudson][hudson]).
> >  
> > +This is a fork from go2xunit by Miki Tebeka <miki.tebeka@gmail.com>.
>
> I'll remove these when merging :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tebeka/go2xunit/pull/20/files#r32376588>.
>